tests/system/Validation/CreditCardRulesTest.php

```diff
@@ -148,46 +148,6 @@ public function creditCardProvider()
                 '6011 0009 9013 9424',
                 true,
             ],
-            'jcb1' => [
-                'jcb',
-                '3530 1113 3330 0000',
-                true,
-            ],
```

Comment:
this possibly need a different key if the value is different?

Reply:
Since these are from the data provider, we can just remove the string keys to use numeric keys instead, or change these duplicate keys to use different names

Reply:
implemented.

Reply:
Sorry, I missed this conversation originally!

There is some merit to keeping the keys (see my top-level comment) but they will need to be deduplicated. The names don't really matter very much, I'm fine with @samsonasik 's proposal of just tacking a digit on the end.